sidebars.js

```diff
@@ -644,22 +644,7 @@ const sidebars = {
         {type: "doc", label: "Flux", id: "articles/flux-integration"},
         {type: "doc", label: "CDEvents", id: "articles/cd-events"},
         {type: "doc", label: "Tekton", id: "articles/tekton"},
-        {
-          type: "category",
-          label: "Slack Integration",
-          items: [
-            {
-              type: "doc",
-              label: "With Webhooks",
-              id: "articles/slack-integration-webhooks",
-            },
-            {
-              type: "doc",
-              label: "With Helm",
-              id: "articles/slack-integration",
```

Comment:
suggest to 
- change the name of this doc to slack-integration
- remove the old slack-integration doc
- add an entry to redirects.js that redirects from slack-integration-webhooks to slack-integration

Reply:
@olensmar , question, do we want to keep the old doc moved to archived in a different path in case it is needed by any of the existing customers using it? Or better to remove it completely?

I was thinking better to keep it but as "archived"&"deprecated", not removing it completely.

Reply:
good idea - we could add it to the "Legacy Documentation" section and add a note at the top that this approach has been deprecated with a link to the new doc. In that case no redirect or delete/renames are needed

Reply:
Done :)

Reply:
@jfermi please see @jmorante-ks suggestion above - let's keep the old integration doc and move it to the "Legacy Documentation" section in the menu, and add a warning at the top saying that this approach has been deprecated in favor of the webhooks-based approach

Reply:
Done